shopfloor_batch_automatic_creation/actions/picking_batch_auto_create.py

```diff
@@ -108,9 +112,12 @@ def gt(value1, value2, digits):
             """Return True if value1 is greater than value2"""
             return tools.float_compare(value1, value2, precision_digits=digits) == 1
 
-        total_weight = 0.0
-        total_volume = 0.0
-        for picking in pickings:
+        total_weight = self._picking_weight(first_picking)
+        total_volume = self._picking_volume(first_picking)
+        for picking in pickings[1:]:
```

Comment:
@sebalix Pay attention to Index error if pickings does not contain more than 1 picking

Reply:
the slice won't raise any error
```py
>>> [][1:]
[]
```

Reply:
@jbaudoux I don't think it's an issue when using slice:
```python
>>> a = self.env.user
>>> a
res.users(1,)
>>> a[1:]
res.users()
>>> b = self.env.user.browse()
>>> b
res.users()
>>> b[42:]
res.users()
```

Reply:
perfect then ;)